server/service/src/sync/sync_api_v5.rs

```diff
@@ -107,108 +127,116 @@ impl SyncApiV5 {
         }
     }
 
-    // Initialize remote sync queue.
-    //
-    // Should only be called on initial sync or when re-initializing an existing data file.
-    pub async fn post_initialise(&self) -> Result<RemoteSyncBatchV5, SyncConnectionError> {
-        let url = self.server_url.join("/sync/v5/initialise")?;
-        // Server rejects initialization request if no `content-length` header included.
-        let mut headers = self.headers.clone();
-        headers.insert(CONTENT_LENGTH, "0".parse().unwrap());
-
-        let request = self
+    pub(crate) fn create_post<T>(
```

Comment:
this is now used, wanted it for a post to upsert end point, and decided to dry things up a little bit (similar for get)